### ISSUE TITLE
Token Grabber

### ISSUE THREAD

**Issue by dh3b** [KSCHdsc/Discord-Token-Grabber#1]
Recently someone logged into my account, I used the grabber a few times. I do not recommend to download this token grabber. It also logged into my friends account, which I tested the grabber with. It took 300$ from his paypal account, buying nitro...

**Comment by wyv3rnnn** [comment]
Yep you can see in the code that it sends the info to a webpage before sending it to your discord webhook.
